### PR TITLE
Patch Istio chart to v1.6.13

### PR DESCRIPTION
**Pull request by shaneutt**
**What type of PR is this?**
Bug

**What this PR does/ why we need it**:
Fixes multiple bugs in our Istio addon

**Which issue(s) this PR fixes**:
https://jira.d2iq.com/browse/D2IQ-70586

**Special notes for your reviewer**:
https://istio.io/latest/news/releases/1.6.x/announcing-1.6.13/

**Does this PR introduce a user-facing change?**:
```release-note
- Fixed an issue where Istiod’s cacert.pem was under the testdata directory (Issue #27574)
- Fixed Pilot agent app probe connection leak. (Issue #27726)
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [x] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

**Review by GoelDeepak — APPROVED** [review]

File: staging/istio/files/crd.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (joejulian) [inline staging/istio/files/crd.yaml:1]: Are we abandoning future upgrades from konvoy < 1.6?

File: staging/istio/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
```
Comment (joejulian) [inline staging/istio/Chart.yaml:1]: If we stick with `apiextensions.k8s.io/v1`, this should be switched to v2 as tiller does not support this.

File: staging/istio/files/crd.yaml (hunk 1 above)
Comment (shaneutt) [reply staging/istio/files/crd.yaml:1]: Yes, Istio is still a staging chart (and a preview addon) expected to land in a post Konvoy 1.6 world as supported.

File: staging/istio/Chart.yaml (hunk 2 above)
Comment (shaneutt) [reply staging/istio/Chart.yaml:1]:
```suggestion
apiVersion: v2
```

Comment (shaneutt) [reply staging/istio/Chart.yaml:1]: :+1:

**Comment by shaneutt** [comment]
Looks like V2 broke how the requirements work. I'm gonna put this in WIP to get that sorted out soon, can't merge yet.

**Comment by shaneutt** [comment]
Due to the 1.6 release and other priorities, this is shelved for now.

**Pull request closed without merging**